src/services/webclient.ts

```diff
@@ -1002,9 +997,6 @@ export class WebClientService {
                         case 'file_too_large':
                             errorMessage = this.$translate.instant('error.FILE_TOO_LARGE');
                             break;
-                        case 'blocked':
-                            errorMessage = this.$translate.instant('error.CONTACT_BLOCKED');
-                            break;
```

Comment:
This condition must not be removed. Now you can drag'n drop a file into a chat view, the app return the "blocked" error.... please re-enable the condition (and the translations)